src/main/resources/com/nec/cyclone/cpp/cyclone/transfer-definitions.hpp

```diff
@@ -107,16 +107,157 @@ struct NullableScalarVec {
 };
 
 // Explicitly instantiate struct template for int32_t
-typedef NullableScalarVec<int32_t> nullable_int_vector;
+//typedef NullableScalarVec<int32_t> nullable_int_vector;
```

Comment:
JavaCPP can use those just fine, no need to copy/paste manually the template:
https://github.com/bytedeco/javacpp/wiki/Mapping-Recipes#creating-instances-of-c-templates